### ISSUE TITLE
variable $footer-padding not used in footer.sass

### ISSUE THREAD

**Issue by jormun-pluxml** [jgthms/bulma#2219]
This is about **Bulma**.

It is a bug in sass file

### Overview of the problem

Doc mention a variable $footer-padding, but padding in footer.sass file is hardcoded.

Fixed footer.sass :
```
$footer-background-color: $white-bis !default
$footer-padding: 3rem 1.5rem 6rem !default
.footer
  background-color: $footer-background-color
  padding: $footer-padding
```

**Comment by jgthms** [comment]
Coming soon.

**Comment by mikehudak** [comment]
why is this closed ? still not fixed! version 0.7.2
